charts/common/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: common
 description: Function library for k8s-at-home charts
 type: library
-version: 2.4.0
+version: 2.4.1
```

New feature, so this would be:
```suggestion
version: 2.5.0
```